test/builder-api/test.sh

```diff
@@ -60,9 +57,7 @@ clean_test_artifacts() {
   sql=
 
   for origin in "${origins[@]}"; do
-    sql+="SET SEARCH_PATH TO shard_0;"
     sql+="DELETE FROM busy_workers WHERE job_id IN (SELECT id FROM jobs WHERE project_name LIKE '$origin%');"
-    sql+="DELETE FROM graph_packages WHERE ident LIKE '$origin%';"
```

Comment:
Why was this line removed?

Reply:
we deleted the table it was referencing https://github.com/habitat-sh/builder/pull/606/files#diff-5e7bf13e8c37dbad84f4d6524185aacbL332